README.md

```diff
@@ -1 +1,4 @@
 # wg21
+
+ [![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/twon/wg21?svg=true&branch=dev_twon_appveyor)](https://ci.appveyor.com/project/twon/wg21)
```

Comment:
This currently points at my Appveyor (pointing at my Github instance of the wg21 repo).  However, before this is checked in then this should be move over to point at Bob's Appveyor instance: https://ci.appveyor.com/api/projects/status/github/bobsteagall/wg21

Reply:
I've updated this now to point at Bob's instance of Appveyor.